src/NUnitFramework/framework/Attributes/ExplicitAttribute.cs

```diff
@@ -62,8 +62,8 @@ public ExplicitAttribute(string reason)
         /// </summary>
         /// <param name="test">The test to modify</param>
         public void ApplyToTest(Test test)
-        {
-            if (test.RunState != RunState.NotRunnable)
+        {
+            if (test.RunState != RunState.NotRunnable && test.RunState != RunState.Ignored)
```

Comment:
Do you think this is the right behavior? I think that if a developer marks a test Explicit and Ignore, then they should be able to run the test if they explicitly set it to run.

Maybe I'm wrong, what do other people think?